spec/Airport_spec.rb

```
@@ -0,0 +1,23 @@
+require "Airport"
+describe Airport do
+  it "lands the plane at the airport" do
+    port = Airport.new
+    plane1 = Plane.new
+    port.land(plane1)
+    expect(port.airport).to include(plane1)
+  end
+  it "checks if the plan took off" do
```

Maybe raise an error when calling take_off method if the @airport array is empty?